### DISCUSSION TITLE
pull request to change address bar endpoint postID to postTitle

### DISCUSSION THREAD

**Discussion by wishrohitv** [DogukanUrker/flaskBlog#111] [Ideas]
address bar with postid
![image](https://github.com/user-attachments/assets/a16641a0-cccc-40e8-a2ac-17d9f13f2c8c)

here is applied version
address bar with post title

https://github.com/user-attachments/assets/dbffb3a1-ea52-4271-9631-86fda0798c38

dear DogukanUrker 
if you agree with to this, i want to create pull request

**Comment by DogukanUrker** [comment]
It looks perfect! Create a PR, and I'll review the code. @wishrohitv

**Reply by wishrohitv** [reply]
Thanks I will create PR
